src/plugins/intel_gpu/src/graph/impls/ocl/broadcast.cpp

```diff
@@ -62,7 +62,7 @@ struct broadcast_impl : typed_primitive_impl_ocl<broadcast> {
         auto output_rank = output_pshape.size();
 
         if (primitive->axes_mapping.empty()) {
-            input_pshape = extend_shape_to_rank_from_begin(input_pshape, output_rank);
+            input_pshape = extend_shape_to_rank_from_end(input_pshape, output_rank);
```

Comment:
@Lyamin-Roman Could you review it? I'm not sure but extend_shape_to_rank_from_begin(..) was intented? Other cases uses extend_shape_to_rank_from_end(..)
Because of this, fc_fp16_eltwise_add unit-tests are failed with broadcast case. (Previous broadcast behavior inserts from end)
And I also change broadcast_shapes_with_params in canonicalization broadcast unit-test.